main.py

```diff
@@ -0,0 +1,45 @@
+from tkinter import Tk, Label, Entry, Frame, StringVar
+
+from technician import secteurs, tech
+
+
+def get_technicians_in_areas(area):
+    return tech.get(area)
+
+
+def get_area_name(area_number):
+    for name, area_numbers in secteurs.items():
+        if area_number in area_numbers:
+            return name
+
+    # nothing found
+    return None
+
+
+def on_user_input(*_):
+    value = user_input.get()
+    if value not in [None, ""]:
+        area_name = get_area_name(int(value))
+        techs = get_technicians_in_areas(area_name)
+
+        output_text =  f"Secteur {area_name}:\n{techs}"
+        output_label.config(text=output_text)
+
+
+# Créer une instance de la fenêtre principale
+app = Tk()
+app.title("DEPARTEMENT O MATIC")
+user_input = StringVar()
+user_input.trace_add("write", on_user_input)

```

Comment:
See [TKinter Entry's Watching for Changes](https://tkdocs.com/tutorial/widgets.html#entry)